versions.tf

```
@@ -2,7 +2,7 @@ terraform {
   required_version = ">= 0.12.31"
 
   required_providers {
-    aws    = "~> 3.35"
+    aws    = ">= 3.35"
```

Could we add a more restricted constraint, we don't know if `5.0` will have breaking changes that will affect this module, what we have done in other modules is https://github.com/lacework/terraform-gcp-config/blob/main/versions.tf#L5
```suggestion
    aws    = ">= 3.35.0, < 5.0.0"
```